src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,6 +12,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_GENDER = new Prefix("gender/");
     public static final Prefix PREFIX_UNIVERSITY = new Prefix("university/");
+    public static final Prefix PREFIX_JOBID = new Prefix("id/");
```

Comment:
I'm not sure if this will cause a conflict for other types of id, maybe we need to be more specific about what id we are specifying.